stan/math/opencl/prim/size.hpp

```diff
@@ -9,13 +9,13 @@ namespace math {
 
 /**
  * Returns the size (number of the elements) of a `matrix_cl`.
- * @param a `matric_cl` to determine size of
+ * @param m `matric_cl` to determine size of
```

Comment:
```suggestion
 * @param m `matrix_cl` to determine size of
```

Reply:
Yeah given how busy Jenkins is lately I will put this in another PR.